tests/validation/tests/v1_api/resource/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ""
+  annotations:
+    field.cattle.io/containerDefaultResourceLimit: { "limitsCpu": "0m","limitsMemory": "0Mi","requestsCpu": "0m","requestsMemory": "0Mi" }
```

Comment:
Do we want to add this as default annotations here? Or is there a reason why we have this? I dont see this yaml file being used anywhere in this PR.

Reply:
this is remveod.